test/Microsoft.AspNet.Mvc.Core.Test/DefaultAssemblyProviderTests.cs

```diff
@@ -98,6 +100,51 @@ public void CandidateAssemblies_ReturnsEmptySequenceWhenReferenceAssembliesIsNul
             Assert.Empty(nullProviderCandidates.Select(a => a.Name));
         }
 
+        // This test verifies DefaultAssemblyProvider.ReferenceAssemblies returns the assembly list that we intended.
+        // If ReferenceAssemblies needs to be changed, the expected list in this test should be updated together.
+        [Fact]
+        public void ReferenceAssemblies_ReturnsExpectedReferenceAssemblies()
```

Comment:
Can this test ever possibly catch a bug? What change could someone be making where this test would provide any meaningful information? I feel like adding a comment to the product source code would be just as useful because there's nothing "computed" here - it's just verifying a static list.